src/public/meeting.ts

```diff
@@ -110,6 +110,13 @@ export namespace meeting {
      */
     isAppSharing: boolean;
   }
+  export interface ISpeakingState {
+    /**
+     * indicates whether one or more participants in a meeting are speaking, or
```

Comment:
NIT: Capitalize "indicates" --> "Indicates"; these comments automatically end up in our public docs so it's good to make them consistent

Reply:
Hi @ydogandjiev thank you for the feedback! I noticed other interfaces in this file more often start with lowercase and was following that convention -- should the others be capitalized too?